scripts/setup-env-macos.sh

```diff
@@ -1,31 +1,26 @@
 #!/bin/bash
 set -x
 
-# Quick install script for mac
-
-# Install homebrew
+# Install Homebrew.
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 
-# Brew install all packages
+# Brew install all packages.
 brew upgrade
 
 brew install \
-	go \
-	libomp \
-	node \
-	git \
-	pyenv \
-	yarn
+  go \
+  libomp \
+  node \
+  git \
+  pyenv \
+  yarn
 
 brew cask install \
-	docker
+  docker
 
-# Install Python 3.6 and virtualenv
 pyenv install 3.6.9
-python3 -m pip install virtualenv
 
-# Set up go mono repo
-mkdir -p "$HOME/go"
-mkdir -p "$HOME/go/bin"
-mkdir -p "$HOME/go/pkg"
-mkdir -p "$HOME/go/src"
+# Set up Determined virtualenv.
+eval "$(pyenv init -)"
+pyenv version 3.6.9
+python3.6 -m venv ~/.virtualenvs/determined
```

Comment:
non-blocking: maybe we shouldn't create a virtualenv?

People like their virtualenvs in different places. Some ppl like to put them in the root of the project, some people in a central location. I think that these scripts should install all the necessary tools, but it isn't necessary to create the venv